sdk/storage/Azure.Storage.Queues/src/Azure.Storage.Queues.csproj

```diff
@@ -19,6 +19,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="System.Text.Json" />
+    <PackageReference Include="Azure.Core.Experimental" VersionOverride="0.1.0-preview.4"/>
```

Comment:
Do we need to remove this before merging?

Reply:
Before merging to master. Not in this PR.